cmd/ebrelayer/main.go

```diff
@@ -365,10 +365,10 @@ func RunInitWitnessCmd(cmd *cobra.Command, args []string) error {
 func replayEthereumCmd() *cobra.Command {
 	//nolint:lll
 	replayEthereumCmd := &cobra.Command{
-		Use:     "replayEthereum [tendermintNode] [web3Provider] [bridgeRegistryContractAddress] [validatorMoniker] [fromBlock] [toBlock] [sifFromBlock] [sifEndBlock]",
+		Use:     "replayEthereum [tendermintNode] [web3Provider] [bridgeRegistryContractAddress] [validatorMoniker]",
 		Short:   "replay missed ethereum events",
-		Args:    cobra.ExactArgs(8),
-		Example: "replayEthereum tcp://localhost:26657 ws://localhost:7545/ 0x30753E4A8aad7F8597332E813735Def5dD395028 validator 100 200 100 200 --chain-id=peggy",
+		Args:    cobra.ExactArgs(4),
+		Example: "replayEthereum tcp://localhost:26657 ws://localhost:7545/ 0x30753E4A8aad7F8597332E813735Def5dD395028 validator --chain-id=peggy",
```

Comment:
Please add the --from and --symbol_translator path/to/file to this command as --from is needed and symbol translation is needed when we are moving bridged assets from ethereum to sifchain and vise versa.